descriptors/ModuleDescriptor-template.json

```diff
@@ -85,12 +85,7 @@
       ]
     }
   ],
-  "requires": [
-    {
-      "id": "invoice-storage.invoices",
```

Comment:
Please correct the interface name to `invoice-storage.invoice`. Please refer to [ModuleDescriptor-template.json](https://github.com/folio-org/mod-invoice-storage/blob/master/descriptors/ModuleDescriptor-template.json#L6) from mod-invoice-storage

Reply:
After offline discussion on Slack, decided to change the interface name 
`invoice-storage.invoice` -> `invoice-storage.invoices`
`invoice-storage.invoice-line` -> `invoice-storage.invoice-lines` 
in mod-invoice-storage repository